cnd/src/storage.rs

```diff
@@ -62,18 +62,18 @@ impl Storage {
 }
 
 #[async_trait::async_trait]
-impl Load<http_api::Swap> for Storage {
-    async fn load(&self, swap_id: LocalSwapId) -> anyhow::Result<http_api::Swap> {
+impl Load<http_api::DecisionSwap> for Storage {
+    async fn load(&self, swap_id: LocalSwapId) -> anyhow::Result<http_api::DecisionSwap> {
         self.db.load_meta_swap(swap_id).await
```

Comment:
Still got the terminology of `meta` here.

Reply:
https://github.com/comit-network/comit-rs/pull/2796